vcontrol/rest/providers/list_all.py

```diff
@@ -8,15 +8,21 @@ class ListProvidersR:
     This endpoint lists all of the providers that have been added.
     """
     allow_origin, rest_url = get_allowed.get_allowed()
+
     def GET(self):
         """ GET HTTP Request """
-        web.header('Access-Control-Allow-Origin', self.allow_origin)
+        try:
+            web.header('Access-Control-Allow-Origin', self.allow_origin)
+        except Exception as e:
+            print(e.message)
         try:
             providers = {}
-            if os.path.isfile('providers.txt'):
-                with open('providers.txt', 'r') as f:
+            providers_file_path = os.path.join(os.path.abspath(__file__), '../providers.txt')
+            if os.path.isfile(providers_file_path):
```

Comment:
both of the asserts that are failing show that the output of `GET()` is `{}` which means that it's passing through this if statement and just returning `providers` as `{}`, so think that means that the path doesn't exist on the test framework.  somehow the way you're creating it with relative paths is not the same on circleci as how you're running it locally.